docs/UserGuide.md

```diff
@@ -132,6 +142,8 @@ You can then access the food item later on for editing, deleting etc.
 **Format:** `add d/DESCRIPTION e/EXPIRY_DATE q/QUANTITY [p/PRIORITY] [t/TAG]…`
 
 * A food item with the same description, expiry date and tags as another food item is considered a duplicate.
+* Description cannot contain more than 50 characters, including spaces.
+* Tags cannot contain more than 72 characters, including spaces.
```

Comment:
Perhaps can say "One tag cannot contain..." instead to be clearer because the restriction is for 1 tag?

Reply:
Thanks!